index.html

```diff
@@ -146,175 +108,158 @@ <h2 class="section-title">Speakers</h2>
+              <span class="name">Lorenzo Peña
+              </span>
+                <a href="https://twitter.com/lorinkoz" target="_blank" class="social-handle">lorinkoz</a>
+              <span class="meta">Of Django, PostgreSQL schemas, and your multi-million dollar idea
+
+              </span>
             </div>
             <p class="description">
-              The way to your first open source pull request leads through many
-              implicit conventions and tools you’re expected to learn. Let’s
-              look at what you’ll need for your first Django contribution,
-              where to find help, and what challenges you’re likely to
-              encounter.
+             In this talk, we'll cover the major challenges of doing multi-tenancy in Django through PostgreSQL schemas. Come along to discover whether all those people are wrong (or not), and whether you'd be de facto wrong (or not) for taking this path.
             </p>
           </li>
 
+        </ul>
+
+      </div>
+    </div>
+
+
+    <!-- Organisers! -->
+    <div class="speakers" id="speakers">
+      <div class="wrap">
+
+        <h2 class="section-title">Organisers</h2>
+
+        <ul class="speaker-list group">
+
           <li class="speaker">
             <div class="img-container">
-              <img src="./images/speakers/melanie-crutchfield.jpg" alt="" />
+              <img src="./images/speakers/markus-holtermann.jpg" alt="Markus Holtermann" />
             </div>
             <div class="details">
-              <span class="name">Melanie Crutchfield</span>
-              <a href="https://twitter.com/HelloMelanieC" target="_blank" class="social-handle">HelloMelanieC</a>
-              <span class="meta">The Unspeakable Horror of Discovering You Didn't Write Tests</span>
+              <span class="name">
+                Markus Holtermann
+              </span>
+                <a href="https://twitter.com/m_holtermann" target="_blank" class="social-handle">m_holtermann</a>
+              <span class="meta">
+                DjangoCon AU Co-Organiser
+              </span>
             </div>
-            <p class="description">
-              I didn’t write tests. I’m here to help you avoid making the same mistake.
-            </p>
           </li>
-
           <li class="speaker">
             <div class="img-container">
-              <img src="./images/speakers/tom-eastman.jpg" alt="" />
+              <img src="./images/speakers/katie-mclaughlin.jpg" alt="Katie McLaughlin" />
             </div>
             <div class="details">
-              <span class="name">Tom Eastman</span>
-              <a href="https://twitter.com/tveastman" target="_blank" class="social-handle">tveastman</a>
-              <span class="meta">Using Python, Django, and ruthlessness to protect people from social media harassment.</span>
+              <span class="name">
+                Markus Holterman
```

Comment:
I think you deserve to have your own name here

Reply:
You sure? Ok.